### PR TITLE
issue-60-fix-chocolatey-download-and-install_failures

### PR DESCRIPTION
**Pull request by ravicharan07**

### CODE REVIEW

File: scripts/install-lansa-base.ps1 [hunk 1]
```diff
@@ -36,7 +36,7 @@ param (
 Write-Debug "script:IncludeDir = $script:IncludeDir" | Write-Host
 
 function ChocoWait([int] $WaitTimeSeconds = 60) {
-    Write-Host "$(Log-Date) Adding Wait for Choco" | Out-Default
+    Write-Host "$(Log-Date) Adding Wait for Choco" | Write-Host
```
Comment (robe070) [inline scripts/install-lansa-base.ps1]: `| Write-Host` must be removed

File: scripts/install-lansa-base.ps1 [hunk 2]
```diff
@@ -96,7 +96,7 @@ try
     }
 
     if ( !(test-path $TempPath) ) {
-        New-Item $TempPath -type directory -ErrorAction SilentlyContinue | Out-Default | Write-Host
+        New-Item $TempPath -type directory -ErrorAction SilentlyContinue | Write-Host
```
Comment (robe070) [inline scripts/install-lansa-base.ps1]: This should not have been changed

File: scripts/install-lansa-base.ps1 [hunk 3]
```diff
@@ -185,7 +185,7 @@ try
 
         # Expand-Archive $installer_file -DestinationPath $InstallerDirectory -Force | Write-Host
 
-        Write-GreenOutput( "$(Log-Date) Unzipping $installer_file to $InstallerDirectory") | Write-Host
+        Write-GreenOutput( "$(Log-Date) Unzipping $installer_file to $InstallerDirectory") | Out-Default | Write-Host
```
Comment (robe070) [inline scripts/install-lansa-base.ps1]: Write-GreenOutput is already formatted correctly. So remove this change.

File: scripts/install-lansa-base.ps1 [hunk 4]
```diff
@@ -258,7 +258,7 @@ try
         # Run-ExitCode 'choco' @( 'install', 'adobereader', '-y', '--no-progress', '--%', '-ia', 'LANG_LIST=en_US' )  | Out-Host
 
         # Stop using Adobe Reader because it was dependent on a Windows Update that could not be installed on Win 2012 because it was obsolete.
-        Run-ExitCode 'choco' @( 'install', 'foxitreader', '-y', '--no-progress' )  | Out-Host
+        Run-ExitCode 'choco' @( 'install', 'foxitreader', '-y', '--no-progress', '-s choco' )  | Out-Host
```
Comment (robe070) [inline scripts/install-lansa-base.ps1]: Replace `| Out-Host` with `| Write-Host`

File: scripts/installGit.ps1 [hunk 5]
```diff
@@ -32,7 +32,7 @@ try {
     if ( $InstallGit -and (-not (Test-Path $GitRepoPath) ) )
     {
         Write-Output "Installing Git" | Out-Host
-        Run-ExitCode 'choco' @('install', 'git', '-y', '--no-progress', '--force' ) | Out-Host
+	Run-ExitCode 'choco' @('install', 'git', '-y', '--no-progress', '--force', '-s choco' ) | Out-Host
```
Comment (robe070) [inline scripts/installGit.ps1]: Replace `| Out-Host` with `| Write-Host`

File: scripts/installGit.ps1 [hunk 6]
```diff
@@ -32,7 +32,7 @@ try {
...
         refreshenv | Out-Host
```
Comment (robe070) [inline scripts/installGit.ps1]: Replace `| Out-Host` with `| Write-Host`

**Review by robe070 — CHANGES_REQUESTED** [review]
Make changes as indicated against each line of code

File: scripts/install-lansa-base.ps1 (hunk 1 above)
Comment (ravicharan07) [reply scripts/install-lansa-base.ps1]: Suggested changes are updated here is the commit  [9385d1eb31d5402e962db9fd51773ad0d152ca52](https://github.com/robe070/cookbooks/pull/95/commits/9385d1eb31d5402e962db9fd51773ad0d152ca52)

**Review by robe070 — CHANGES_REQUESTED** [review]
Line 184 New-Item $InstallerDirectory -ItemType directory -Force
needs 
`| Out-Default | Write-Host`

Line 198 destinationFolder.CopyHere($zipFile.Items(), $copyFlags)
needs 
`| Out-Default | Write-Host`

line 207 .\amazon-cloudwatch-agent-ctl.ps1 -a start -s
needs 
`| Out-Default | Write-Host`

**Comment by ravicharan07** [comment]
@robe070 recommended changes has been adjusted. Please re-review.

**Review by robe070 — APPROVED** [review]

**Pull request merged**